lib/Rex/Fork/Manager.pm

```diff
@@ -27,18 +27,20 @@ sub new {
 }
 
 sub add {
-  my ( $self, $task, $start ) = @_;
```

Comment:
I think `$start` is being used in (Rex::TaskList::Base#L317)[https://github.com/RexOps/Rex/blob/master/lib/Rex/TaskList/Base.pm#L317]. Can you please add it back?